### PR TITLE
Make start/finish_interaction log messages and store corresponding clock levels in the event sent to the MCP.

### PR DESCRIPTION
**Pull request by itamarst**

### CODE REVIEW

File: quark/metrics.q [hunk 1]
```diff
@@ -40,6 +40,12 @@ namespace mdk_metrics {
         @doc("Map destination node UUID to success=1/failure=0 of interaction.")
         Map<String,int> results = {};
 
+        @doc("Causal clock level at start.")
```
Comment (plombardi89) [inline quark/metrics.q:43]: Can you fill me in on how I am supposed to store startClock[] and endClock[]

The DB has a single clock[] column on the interaction table. Does this need to be changed?

File: quark/mdk-2.0.q [hunk 2]
```diff
@@ -788,6 +788,12 @@ namespace mdk {
         }
 
         void start_interaction() {
```
Comment (plombardi89) [inline quark/mdk-2.0.q:790]: Public API? Why underscore vs camelcase... I wish Quark could just "do the right thing" and generate idiomatic API's across languages.

**Review by plombardi89 — COMMENTED** [review]
Apparently I need to "submit" the comments twice in the GH UI?

File: quark/metrics.q (hunk 1 above)
Comment (itamarst) [reply quark/metrics.q:43]: Yeah, you'd want to add another column. We want to store both the clock level of the start of the interaction and the clock level at the end of the interaction so we correlate with logs on both start and end.

File: quark/mdk-2.0.q (hunk 2 above)
Comment (itamarst) [reply quark/mdk-2.0.q:790]: It was like that when I got here :( We've discussed plans for having Quark do that, yeah.

**Review by plombardi89 — APPROVED** [review]
OK

**Pull request merged**